config/v1/types_infrastructure.go

```diff
@@ -614,6 +625,15 @@ type OpenStackPlatformStatus struct {
 	// datacenter DNS, a DNS service is hosted as a static pod to serve those hostnames
 	// to the nodes in the cluster.
 	NodeDNSIP string `json:"nodeDNSIP,omitempty"`
+
+	// loadBalancerType defines if a internal or external load balancer
+	// is to be used for the openshift api and ingress endpoints.
+	// When set to Internal the static pods in charge of API and Ingress traffic load-balancing
+	// defined in machine config operator will be deployed.
+	// When set to External these static pods will not be deployed.
+	// When not set, the default value is left to the cluster and currently defaults
+	// to Internal, so the static pods will be deployed.
+	LoadBalancerType PlatformLoadBalancerType `json:"loadBalancerType,omitempty"`
```

Comment:
We're defining this in the Status, and only for OpenStack.

Which controller sets this field?

Do we want this to be common for all providers, or only OpenStack?

Reply:
I'll add it to some other platforms (BM and vsphere for now).

It's set in the installer, like other PlatformStatus fields.

Reply:
I'm closing it because Ben replied in the enhancement, saying for now this should be in the `PlatformStatus`. Happy to discuss this further if this needs further debate.